app/full_text_search.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import re
+
+from google.appengine.api import search as appengine_search
+
+import model
+
+# The index name for full text search
+PERSON_FULLTEXT_INDEX_NAME = 'person_information'
+
+def search(repo, query_obj, max_results):
+    """
+    Searches person with index.
+    Args:
+        repo: The name of repository
+        query_obj: Search word
+        max_results: The max results you want.(Maximum: 1000)
+
+    Returns:
+        results[<model.Person>, ...]
+
+    Raises:
+        search.Error: An error occurred search unknown index, syntax error
```

Comment:
What do you mean? Do you mean "An error occurred when the index name is unknown or the query has a syntax error"?